buildscript/sbom.go

```diff
@@ -0,0 +1,10 @@
+package buildscript
+
+import (
+	"github.com/craiggwilson/goke/pkg/sh"
+	"github.com/craiggwilson/goke/task"
+)
+
+func WriteSBOMLite(ctx *task.Context) error {
+	return sh.Run(ctx, "scripts/regenerate-sbom.sh")
```

Comment:
Why don't we write this as Go code?

Reply:
If you look at the script you'll see it has a lot of pipes. We'd have to reimplement that all in Go. I just copied most of this from our docs for the `silkbomb` tool. See https://docs.devprod.prod.corp.mongodb.com/mms/python/src/sbom/silkbomb/docs/PURLS#golang-purl-example